main/src/cgeo/geocaching/connector/trackable/TrackableConnector.java

```diff
@@ -19,6 +19,8 @@
  */
 public interface TrackableConnector {
 
+    public int getPreferenceActivity();
```

Comment:
When extending interfaces, it is a good idea to add javadoc. We have so little javadoc (meaning we didn't do a good job at this in the past), and it is hard to understand for other developers what is meant with "isRegistered", "isActive", ... etc.